pkgs/development/python-modules/optimum/default.nix

```
@@ -39,6 +45,10 @@ buildPythonPackage rec {
 
   build-system = [ setuptools ];
 
+  pythonRelaxDeps = [
+    "transformers"
+  ];
```

```suggestion
  pythonRelaxDeps = [ "transformers" ];
```
please respect `nixfmt-rfc-style`.

---

I forgot to run the formatter after my change.
Good catch !